esm/index.js

```diff
@@ -83,11 +83,19 @@ const app = Class ? Class.app : freeze({
   }
 });
 
+function isModifiedEvent(event) {
```

Comment:
mind changing this as:

```js
const isModifiedEvent = ({metaKey, altKey, ctrlKey, shiftKey}) => !!(metaKey || altKey || ctrlKey || shiftKey);
```

thanks!

Reply:
Sure, done

Reply:
I don't see it? 🤔

Reply:
Are you still seeing the previous commit?

https://github.com/WebReflection/a-route/pull/2/files